src/types.rs

```diff
@@ -237,6 +237,12 @@ pub struct RelayPreferredExit {
     pub id: String,
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq)]
+pub enum VpnProvider {
```

Comment:
This enum needs an "Other" option so that old app versions don't crash if we add a new provider.

Reply:
Should I just make this field a string, but then in the backend ensure it's typed? That way the client will not lose information?

Reply:
Okay yeah I think it's better to add this to the backend itself for config parsing, and then convert to string for the response.

Reply:
I'm ok to just have a `provider_name` for now. It would be nice to have some machine-readable field but we can always add that later.

Reply:
Yeah, for the backend DB having an enum is a great idea, then it can just put out a string for the API response.